tests/e2e/tkgs_ha_utils.go

```diff
@@ -898,3 +898,17 @@ func verifyStsVolumeMetadata(client clientset.Interface, ctx context.Context, na
 	}
 
 }
+
+// verifyAnnotationsAndNodeAffinityInSVC verifies annotations on SVC PVC
+// and node affinities and pod location of volumes on correct zones
+func verifyAnnotationsAndNodeAffinityInSVC(allowedTopologyHAMap map[string][]string,
```

Comment:
why we are writing a new util, there is already one created for the same requirement - verifyPVnodeAffinityAndPODnodedetailsForStatefulsetsLevel5

Reply:
modified to use verifyAnnotationsAndNodeAffinity